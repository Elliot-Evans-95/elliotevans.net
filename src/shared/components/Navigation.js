// @flow

import React from 'react';
import { AppNavigation, NavItems, StyledLink } from '../styles/styles';

const Navigation = () => {
  return (
    <AppNavigation>
      <ul>
        <NavItems>
          <StyledLink to="/" exact activeClassName="selected">
            Home
          </StyledLink>
        </NavItems>
        <NavItems>
          <StyledLink to="/about" activeClassName="selected">
            About
          </StyledLink>
        </NavItems>
        <NavItems>
          <StyledLink to="/work" activeClassName="selected">
            Work
          </StyledLink>
        </NavItems>
        <NavItems>
          <StyledLink to="/contact" activeClassName="selected">
            Contact
          </StyledLink>
        </NavItems>
      </ul>
    </AppNavigation>
  );
};

export default Navigation;
