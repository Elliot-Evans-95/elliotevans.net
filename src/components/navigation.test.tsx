import React from 'react';
import renderer from 'react-test-renderer';

import Navigation from './navigation';

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Navigation />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
