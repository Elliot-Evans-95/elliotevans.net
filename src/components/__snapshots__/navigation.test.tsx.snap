// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  className="css-v2ilfb-AppNavigation e1h04ftx7"
>
  <ul>
    <li
      className="css-1o2qtws-NavItems e1h04ftx8"
    >
      <a
        className="css-16crw00-StyledLink e1h04ftx10"
        href="/"
      >
        Home
      </a>
    </li>
    <li
      className="css-1o2qtws-NavItems e1h04ftx8"
    >
      <a
        className="css-16crw00-StyledLink e1h04ftx10"
        href="/about/"
      >
        About
      </a>
    </li>
    <li
      className="css-1o2qtws-NavItems e1h04ftx8"
    >
      <a
        className="css-16crw00-StyledLink e1h04ftx10"
        href="/projects/"
      >
        Projects
      </a>
    </li>
    <li
      className="css-1o2qtws-NavItems e1h04ftx8"
    >
      <a
        className="css-16crw00-StyledLink e1h04ftx10"
        href="/contact/"
      >
        Contact
      </a>
    </li>
  </ul>
</nav>
`;
